client/web/src/components/HeroPage.scss

```
@@ -36,9 +36,9 @@
     }
 
     &__title {
-        font-size: 2em;
+        font-size: 2rem;
         margin-top: 1rem;
-        line-height: 2.5rem;
+        line-height: 1.1111111111;
```

1.11111 * 2rem = 2.2222rem instead of 2.5rem before
```suggestion
        line-height: 1.5;
```